docs/content/_navigation.json

```diff
@@ -50,6 +50,10 @@
           {
             "title": "Testing Assets",
             "path": "/tutorial/assets/testing-assets"
+          },
+          {
+            "title": "Scheduling Assets",
+            "path": "/tutorial/assets/scheduling-assets"
```

Comment:
the other tutorials are more sequential, i.e. they are step-by-step instructions that build on top of the previous page, while this one feels like a guide. should we move this tutorial to guides here https://docs.dagster.io/guides/general, and add a link to the guide here: https://dagster-git-docs-lopp-freshness-tutorial-elementl.vercel.app/tutorial/assets/testing-assets#conclusion?

Reply:
when it's a guide, it's ok to use arbitrary a, b, c as asset names. if we decide to keep this as the last step in the tutorial, naming assets something more meaningful would make more sense.